src/config.ts

```diff
@@ -301,14 +293,7 @@ export class MirrordConfigManager {
             return predefinedConfig;
         }
 
-        let defaultConfig = await MirrordConfigManager.createDefaultConfig(folder);
-        new NotificationBuilder()
-            .withMessage(`created a default mirrord config in folder ${folder.name}`)
-            .withOpenFileAction(defaultConfig)
-            .withDisableAction("promptCreatedDefaultConfig")
```

Comment:
Looks like we're deleting all the uses of that prompt, should we maybe delete it from [`package.json`](https://github.com/eyalb181/mirrord-vscode/blob/638dfa37183666a06cab43d75b7278b63d150b67/package.json#L96), or do you want to keep it there for future use?